modules/foleys_gui_magic/State/foleys_MidiParameterMapper.cpp

```diff
@@ -90,6 +96,20 @@ void MidiParameterMapper::mapMidiController (int cc, const juce::String& paramet
     auto mappings = getMappingSettings();
 
     juce::ValueTree node { IDs::mapping, {{IDs::cc, cc}, {IDs::parameter, parameterID}} };
+
+    // If mapping already there, remove it:
+    if (mappings.isValid()) {
```

Comment:
Sorry for my OCD, but please let's stay with 4 spaces ;-)
I intend to ad clang-format further down, so it is no longer a manual task.

Reply:
I do try to keep to four spaces in your code, but sometimes I slip, because both Emacs and Xcode are set to two spaces for me.  Yes, clang-format is a good answer!